kolibri/plugins/learn/assets/src/views/EmbeddedSidePanel/index.vue

```diff
@@ -135,18 +142,6 @@
     const value = ResourcesNeededTypes[key];
     // TODO rtibbles: remove this condition
     if (plugin_data.learnerNeeds.includes(value) || process.env.NODE_ENV !== 'production') {
-      // For some reason the string ids for these items are in PascalCase not camelCase
-      if (key === 'PEOPLE') {
-        key = 'ToUseWithTeachersAndPeers';
-      } else if (key === 'PAPER_PENCIL') {
-        key = 'ToUseWithPaperAndPencil';
-      } else if (key === 'INTERNET') {
-        key = 'NeedsInternet';
-      } else if (key === 'MATERIALS') {
-        key = 'NeedsMaterials';
-      } else if (key === 'FOR_BEGINNERS') {
-        key = 'ForBeginners';
-      }
```

Comment:
How are you able to delete this? 🤔

Reply:
For the benefit of future reviewers - this was code that had already been ported into the commonCoreStrings mixin, so is now redundant.